third_party/skcms.cmake

```diff
@@ -15,10 +15,15 @@
 add_library(skcms-obj OBJECT EXCLUDE_FROM_ALL skcms/skcms.cc)
 target_include_directories(skcms-obj PUBLIC "${CMAKE_CURRENT_LIST_DIR}/skcms/")
 
+add_library(skcms-interface INTERFACE)
```

Comment:
A comment here about what's this lib would be appreciated by future you. Mention that the skcms-interface is meant to be compiled/used by external project (like the plugins) and not the bundled version in libjxl which uses the wrapper.